packages/contracts-bedrock/src/dispute/FaultDisputeGame2.sol

```diff
@@ -942,4 +947,43 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
             ? Hash.wrap(keccak256(abi.encode(_disputed, _disputedPos)))
             : Hash.wrap(keccak256(abi.encode(_starting, _startingPos, _disputed, _disputedPos)));
     }
+
+    function findPreStateClaim(
```

Comment:
It would be helpful to provide some comments of `findPreStateClaim` and `findPostStateClaim`.